test/threadstest.c

```diff
@@ -82,15 +91,392 @@ static int test_lock(void)
     int res;
 
     res = TEST_true(CRYPTO_THREAD_read_lock(lock))
-          && TEST_true(CRYPTO_THREAD_unlock(lock))
-          && TEST_true(CRYPTO_THREAD_write_lock(lock))
-          && TEST_true(CRYPTO_THREAD_unlock(lock));
+        && TEST_true(CRYPTO_THREAD_unlock(lock))
+        && TEST_true(CRYPTO_THREAD_write_lock(lock))
+        && TEST_true(CRYPTO_THREAD_unlock(lock));
```

Comment:
Why this change? It looks more correct to me the way it was before??

Reply:
because check-format.pl complained about it, and I was in that code so I fixed it, but sure, I'll put it back